html2asketch/base.js

```diff
@@ -29,14 +30,14 @@ class Base {
     this._style = style;
   }
 
-  toJSON() {
+  toJSON(indexedParentID) {
     if (!this._class) {
       throw new Error('Class not set.');
     }
 
-    return {
+    const res = {
       '_class': this._class,
-      'do_objectID': this._objectID,
+      'do_objectID': indexedParentID || this._objectID,
```

Comment:
does Sketch care about IDs of the child layers? I thought it only cares about the top level IDs (symbol, document)

Reply:
i dont know, i installed sketch yesterday

Reply:
maybe there is no need to make child layers deterministic if Sketch doesn't care for that - we should test that